src/natcap/invest/habitat_quality.py

```diff
@@ -64,128 +60,116 @@
+                    "type": "raster",
+                    "bands": {1: {"type": "ratio"}},
+                },
+                "fut_path": {
+                    "required": "lulc_fut_path",
+                    "type": "raster",
+                    "bands": {1: {"type": "ratio"}},
+                }
+            },
             "about": (
-                "A CSV file of all the threats for the model to consider. "
-                "Each row in the table is a degradation source. The columns "
-                "(THREAT, MAX_DIST, WEIGHT, DECAY) are different attributes "
-                "of each degradation source. The columns "
-                "(BASE_PATH, CUR_PATH, FUT_PATH) specify the filepath name "
-                "for the degradation source where the path is relative to "
-                "the THREAT CSV. Column names are case-insensitive. THREAT: "
-                "The name of the threat source and this name must match "
-                "exactly to the name of it's corresponding column in the "
-                "sensitivity table. "
-                "MAX_DIST: A number in kilometres (km) for the maximum "
-                "distance a threat has an affect. WEIGHT: A "
-                "floating point value between 0 and 1 for the threats "
-                "weight relative to the other threats.  Depending on the "
-                "type of habitat under review, certain threats may cause "
-                "greater degradation than other threats. "
-                "DECAY: A string value of either exponential or "
-                "linear representing the type of decay over space for "
-                "the threat. See the user's guide for valid values "
-                "for these columns. "
-                "BASE_PATH: optional. Required if baseline LULC input. "
-                "The THREAT raster filepath for the base scenario where the "
-                "filepath is relative to the THREAT CSV input. Entries can "
-                "be left empty if there is no baseline scenario or if using "
-                "the baseline LULC for rarity calculations only. "
-                "CUR_PATH: required. The THREAT raster filepath for the "
-                "current scenario where the filepath is relative to the "
-                "THREAT CSV input. "
-                "FUT_PATH: optional. Required if threat LULC input. The "
-                "THREAT raster filepath for the future scenario where the "
-                "filepath is relative to the THREAT CSV input. Entries can "
-                "be left empty if looking at current scenario only."
-                ),
-            "name": "Threats Data"
+                "Table mapping each threat of interest to its properties and "
+                "maps of its distribution. The raster columns give filepaths "
+                "to maps of the relative intensity of each threat, ranging "
+                "from 0 to 1. Paths are relative to the threats table path."),
```

Comment:
Would it make sense in this case to have something like:
> Refer to the Users Guide for a detailed example

Reply:
I'm inclined not to, just because it applies to almost every input. We can point out that the user's guide has examples on the Getting Started page or somewhere, and then not have to include/translate that phrase for a whole bunch of args.